charts/castai-pod-pinner/values.yaml

```diff
@@ -37,6 +37,8 @@ serviceAccount:
 
 podAnnotations: {}
 
+podLabels: {}
```

Comment:
nit: since we're generating docs automatically, it would be great to have some description here.